easybuild/easyconfigs/t/typing-extensions/typing-extensions-4.8.0-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'PythonPackage'
+
+name = 'typing-extensions'
+version = '4.8.0'
```

Comment:
We have `typing-extensions` 4.6.3 included in `Python-bundle-PyPI-2023.06-GCCcore-12.3.0.eb`, so we can just use that instead?

Reply:
Sure, but this is still an upgrade anyway, since the latest version of `typing-extensions` we have is `4.3.0-GCC-11.3.0`. Should I create a separate PR for that?

Reply:
We can just remove `typing-extensions-4.8.0-GCCcore-12.3.0.eb` entirely, since `typing-extensions` is now included in `Python-bundle-PyPI`, there isn't a good reason to still have that as a separate easyconfig as well.

Reply:
done in [a96b92e](https://github.com/easybuilders/easybuild-easyconfigs/pull/19091/commits/a96b92e6c481a73c5fe99a5b279f1170436f2ec4)